packages/elements/src/item/index.ts

```diff
@@ -179,14 +157,51 @@ export class Item extends ControlElement {
   };
 
   /**
-   * @override
+   * Handles aria-selected or aria-checked when toggle between single and multiple selection mode
+   * @returns {void}
+   **/
+  private multipleChanged (): void {
+    this.removeAttribute(this.multiple ? 'aria-selected' : 'aria-checked');
+    this.selectedChanged();
+  }
+
+  /**
+   * Handles aria when selected state changes
    * @returns {void}
    */
-  protected update (changedProperties: PropertyValues): void {
+  private selectedChanged (): void {
+    this.setAttribute(this.multiple ? 'aria-checked' : 'aria-selected', String(this.selected));
+  }
+
+  /**
+   * Control State behaviour will update tabindex based on the property
+   * @returns {void}
+   */
+  private typeChanged (): void {
+    const noInteraction = this.type === 'header' || this.type === 'divider' || this.disabled;
+    if (noInteraction) {
+      this.disableFocus();
+    }
+    else if (!this.disabled) {
+      this.enableFocus();
+    }
+  }
+
+  /**
+   * Invoked before update() to compute values needed during the update.
+   * @param changedProperties changed properties
+   * @returns {void}
+   */
+  protected willUpdate (changedProperties: PropertyValues): void {
     if (changedProperties.has('type')) {
       this.typeChanged();
     }
-    super.update(changedProperties);
+    if (changedProperties.has('multiple')) {
+      this.multipleChanged();
+    }
+    if (changedProperties.has('selected')) {
```

Comment:
`multipleChanged` calls `selectedChanged` so use `else if` so as not to execute twice.
```suggestion
    else if (changedProperties.has('selected')) {
```